src/app/_services/api/event-group.service.ts

```diff
@@ -21,4 +22,13 @@ export class EventGroupService extends BaseService<EventGroupsIndexDto> {
       new EventGroupSerializer()
     );
   }
+
+  unlinkEvent(groupId: string, eventId: string): Observable<any> {
+    const endpoint = this.endpoint.ENDPOINT;
+
+    return this.httpClient.post(
+      `${this.helperService.getApiUrl()}/${endpoint}/${groupId}/unlinkEvent?eventUuid=${eventId}`,
```

Comment:
I think we should add the "unlinkEvent" in another atribute in the api.ts file   
ENDPOINT: 'eventGroups',
  GET_ALL: 'eventGroups/indexList',
  UPDATE: 'eventGroups/push',
  ADD: 'eventGroups/push',
  DELETE: 'eventGroups/delete',
  UNLINK: 'eventGroups/unlinkEvent',